plutus-benchmark/plutus-benchmark.cabal

```diff
@@ -557,13 +541,25 @@ test-suite plutus-benchmark-marlowe-tests
 
 -- TODO: Add benchmarks for the executable semantics when we have a UPLC version
 
+library agda-internal
+  import:          lang, turn-off-plugin-warning, ghc-version-support
```

Comment:
Why are you still including `ghc-version-support`? You've added it to the stuff that depends on this, but the right thing is to just _remove_ it: none of these depend on the plugin!

Reply:
> none of these depend on the plugin!

Yes they do! `nofib-agda-cek` and `marlowe-agda-cek` both depend on the plugin and they were getting it from `agda-internal`: they definitely don't work with 8.10.  I'm just about to push a fix that'll make them depend on the plugin directly and then `validation-agda-cek` should work with all versions.  I think there are no unnecessary plugin dependencies in the other tests.

Reply:
It certainly sounds like those two components need to be marked as not buildable on earlier GHCs, but I don't see why they should need to depend on the plugin directly?